style.css

```diff
@@ -467,3 +555,185 @@ hr {
   background: #0d151e;
   border-radius: 10px;
 }
+
+/* screen for- Desktop: 768px or wider */
+@media screen and (min-width: 1024px) {
```

Comment:
[OPTIONAL] Great job implementing queries for the `1024px` media breakpoint, I really believe this design needs one 👍. However, I suggest you start tacking how your website would look from the `768px` query instead since that is the minimum screen size for desktops.

see below 

[screen-recorder-sat-dec-10-2022-09-30-03.webm](https://user-images.githubusercontent.com/84930355/206841329-d186e1e0-c17e-46f9-acee-c9de51a9a52a.webm)


When I put the width at `768px` the page behaves like a mobile. Please address this.